templates/angular/igx-ts/combo/default/files/src/app/__path__/__name__.component.ts

```diff
@@ -0,0 +1,18 @@
+import { ChangeDetectorRef, Component, OnInit, ViewChild } from '@angular/core';
+import { IgxComboComponent  } from 'igniteui-angular';
+import { localData  } from "./local-data";
+
+@Component({
+	selector: 'app-$(filePrefix)',
```

Comment:
Mixed tabs and spaces ⚠️ 
Should be double space as indentation for this project.